eslint-config/package.json

```diff
@@ -8,6 +8,7 @@
   "devDependencies": {
     "@babel/eslint-parser": "^7.17.0",
     "@babel/preset-react": "^7.16.7",
+    "@types/jquery": "^3.5.13",
```

Comment:
I think this belongs to the plugin, not to `eslint-config`.

Reply:
Addressed in 487310f5b633a723ccc8b6cc0eb13a558231f8a6